src/server/controllers/users.controller.js

```diff
@@ -13,8 +13,8 @@ let mockData = {
 };
 
 export function saveTest(req, res, next) {
-  let user = new user(mockData);
-  user.save((err) => {
+  let userSample = new user(mockData);
```

Comment:
I think the method name was confused, it's for inserting mock data.
And user.js is the test for this method.